Demo/Demo/DemoSettings/DemoSettings.swift

```diff
@@ -46,4 +48,14 @@ enum DemoSettings {
             return "TODO"
         }
     }
+
+    /// Added secret to get order from api. This shouldnt be needed
```

Comment:
I'm not familiar with the API. Why shouldn't this be needed and what is it doing? Is it temporary?

Reply:
@mattwylder we need it at the moment because the endpoint `confirm_payment_source` needs it in the auth header. Its temporary for now, because the next step in this integration is to remove and use access tokens instead